src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -1266,35 +1127,41 @@ private sealed class RuntimeMessageBox
             {
                 var message = string.Format(formatProvider, format, args);
 
-                Console.Error.WriteLine(
+                // todo: use the logger here too
+
+                this.logger.Error(
                     string.Concat("[", icon.ToString(), "]: ", message));
```

Comment:
should this just call into the other Show method?

Reply:
We could, but what buttons and caption would be passed? I think this is best addressed as part of issue #109 .